src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/BatchingSqlJournal.cs

```diff
@@ -862,25 +940,24 @@ private void TryProcess()
             {
                 _remainingOperations--;
 
-                var chunk = DequeueChunk(_remainingOperations);
+                var (chunk, isWrite) = DequeueChunk(_remainingOperations);
```

Comment:
`DequeueChunk` automatically separate the read and write requests in the queue, we need to pull the read and write information of the current batch chunk from there.